frontend/src/services/notebookService.ts

```diff
@@ -263,11 +262,8 @@ const enableNotebook = async (notebook: Notebook): Promise<Notebook> => {
     return Promise.reject('Notebook is not assigned to a namespace -- cannot start it');
   }
 
-  return patchNotebook(
-    namespace,
-    name,
-    _.merge({}, notebook, { metadata: { annotations: { 'kubeflow-resource-stopped': null } } }),
-  );
+  notebook.metadata.annotations['kubeflow-resource-stopped'] = undefined;
+  return replaceNotebook(namespace, name, notebook);
```

Comment:
Why not the merge logic before? Directly mutating an object that is not yours is going to cause problems in a functional world. That's why we use an empty object at the start of `_.merge` calls... so no object references are mutated and thus echo'ed back up the call stack.

Reply:
I was playing around with this and ended up with that, will change it.